modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionLocalServiceImpl.java

```diff
@@ -224,7 +224,7 @@ public ObjectDefinition addOrUpdateSystemObjectDefinition(
 					newObjectField.getDBType(),
 					newObjectField.getDefaultValue(), false, false, "",
 					newObjectField.getLabelMap(), newObjectField.getName(),
-					newObjectField.isRequired());
+					newObjectField.isRequired(), newObjectField.getState());
```

Comment:
Hi @seltong,

Could we replace `newObjectField.getState()`to `newObjectField.isState()`?